Template/Variable/MatomoConfigurationVariable.php

```diff
@@ -93,6 +93,7 @@ public function getParameters()
             }),
             $this->makeSetting('enableDoNotTrack', false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
                 $field->title = 'Enable Do Not Track';
+                $field->description = 'Enable this feature to not track users who opt out of tracking using Mozilla\'s (proposed) Do Not Track setting';
```

Comment:
Looks good. @tsteur is there a special reason why all those fields & descriptions are not translatable. Or is that something that might be worth to change in this or another pull request?

Reply:
there's #49

Reply:
Moved to en.json to make it translatable